### DISCUSSION TITLE
Getting rejected while applying for Student Developer Pack

### DISCUSSION THREAD

**Discussion by DanteHaxor** [community/community#57563] [GitHub Education]
### Hi everyone,

Dear GitHub Support,

I am encountering errors while attempting to avail the Student Developer Pack. To confirm my student status, I can send you a mail with my university edu mail and I can attach my university ID card also

I kindly request your assistance in resolving this issue promptly, as it would greatly benefit my academic pursuits. Please let me know if any further information is required.

Thank you for your attention and support.

**Comment by samus-aran** [comment]
Hi @DanteHaxor 👋 

Here are a few tips on submitting an application.

Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor rather than a phone perhaps.
If you can, print off your evidence and take a photo of this copy.
Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.

Let us know how you get on 👍
